frontend/pkg/frontend/ocm_test.go

```diff
@@ -0,0 +1,65 @@
+package frontend
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/Azure/ARO-HCP/internal/api/arm"
+	"github.com/google/uuid"
+)
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
```

Comment:
```suggestion

// Copyright (c) Microsoft Corporation.
// Licensed under the Apache License 2.0.

import (
	"context"
	"io"
	"net/http"
	"net/http/httptest"
	"testing"

	"github.com/Azure/ARO-HCP/internal/api/arm"
	"github.com/google/uuid"
)

```

Reply:
I've followed the current pattern in existing files :)

Reply:
Oh its actually then inconsistent across the entire project :D :D